src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2.java

```diff
@@ -23,29 +23,30 @@
 import java.util.function.Function;
 import java.util.function.Supplier;
 
-public class InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2{ // violation
+public class
+    InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2{ // violation ''{' is not .* whitespace'
```

Comment:
We need more space for message, it is not clear what message is.
Please make this line without violation and create new top level class ( not public) in this file with violation.

Reply:
done